setup.cfg

```diff
@@ -0,0 +1,22 @@
+[flake8]
+ignore = W503, E203, B305
+max-line-length = 88
+
+[mypy]
+disallow_untyped_defs = True
+ignore_missing_imports = True
+
+[mypy-tests.*]
+disallow_untyped_defs = False
+check_untyped_defs = True
+
+[tool:isort]
+profile = black
+combine_as_imports = True
+known_first_party = uvicorn,tests
+known_third_party = click,does_not_exist,gunicorn,h11,httptools,pytest,requests,setuptools,urllib3,uvloop,watchgod,websockets,wsproto
+
+[tool:pytest]
+addopts = --cov=uvicorn --cov=tests -rxXs
+markers =
+  copied_from(source, changes=None): mark test as copied from somewhere else, along with a description of changes made to accodomate e.g. our test setup
```

Comment:
It seems we don't need this marker. :-) (And we should remove it from HTTPX - it's an old leftover.)

Reply:
done